### PR TITLE
LPS-102598

### PR DESCRIPTION
**Pull request by hhuijser**

### CODE REVIEW

**Comment by hhuijser** [comment]
ci:test:relevant

**Comment by hhuijser** [comment]
ci:test:sf

File: modules/core/portal-bootstrap/system.packages.extra.bnd [hunk 1]
```diff
@@ -106,25 +106,25 @@ Export-Package:\
...
+	com.sun.syndication.feed;version='1.0.LIFERAY-PATCHED-1',\
+	com.sun.syndication.feed.atom;version='1.0.LIFERAY-PATCHED-1',\
+	com.sun.syndication.feed.impl;version='1.0.LIFERAY-PATCHED-1',\
+	com.sun.syndication.feed.module;version='1.0.LIFERAY-PATCHED-1',\
+	com.sun.syndication.feed.module.impl;version='1.0.LIFERAY-PATCHED-1',\
+	com.sun.syndication.feed.rss;version='1.0.LIFERAY-PATCHED-1',\
+	com.sun.syndication.feed.synd;version='1.0.LIFERAY-PATCHED-1',\
+	com.sun.syndication.feed.synd.impl;version='1.0.LIFERAY-PATCHED-1',\
+	com.sun.syndication.io;version='1.0.LIFERAY-PATCHED-1',\
+	com.sun.syndication.io.impl;version='1.0.LIFERAY-PATCHED-1',\
```
Comment (rotty3000) [inline modules/core/portal-bootstrap/system.packages.extra.bnd]: Hey @hhuijser these versions are no good. Since these are OSGi package versions which are going to be visible to the framework, they have to be OSGi compatible. chars are only allowed in the qualifier part (which is the fourth element after a third period  `major.minor.micro.qualifier`).

Secondly, adding the qualifier doesn't really help for package versions anyway, so we should never add it. It's just noise and will only lead to errors if the syntax isn't OSGi compatible.

In short, I would omit _all_ qualifiers. That should actually shorten your list of changes significantly.

Comment (hhuijser) [reply modules/core/portal-bootstrap/system.packages.extra.bnd]: Makes sense, will get those changes in. Thanks for the feedback
